coretex/cli/modules/user.py

```diff
@@ -15,16 +15,43 @@
 #     You should have received a copy of the GNU Affero General Public License
 #     along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+import requests
+
 from . import ui
-from ...networking import networkManager, NetworkRequestError
+from ...networking import networkManager, NetworkRequestError, baseUrl
 from ...configuration import UserConfiguration, InvalidConfiguration, ConfigurationNotFound
 
 
+def validateServerUrl(serverUrl: str) -> bool:
+    try:
+        endpoint = baseUrl(serverUrl) + "/info.json"
+        response = requests.get(endpoint, timeout = 5)
+        return response.ok
+    except BaseException as ex:
```

Comment:
`BaseException` should never be catched. In this case using `requests.exceptions.RequestException` is more fitting.